CMakeLists.txt

```diff
@@ -66,7 +66,9 @@ configure_file(
   "${PROJECT_SOURCE_DIR}/src/qcdloop.pc.in"
   "${PROJECT_SOURCE_DIR}/src/qcdloop.pc"
   )
-IF(${CMAKE_SYSTEM_PROCESSOR} MATCHES "x86_64")
```

Comment:
OK, Something like 

```
IF(${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "x86_64|i686|ppc64|ppc64le|ppc|ppcle" )
```
would be better.
But it is important to list supported architectures explicitly, and not "everything but...".

Reply:
@andriish Can you explain more on why? Is there another architecture that supports `float128` that you know of? How is this different or better?